language/move-prover/bytecode/src/borrow_analysis.rs

```diff
@@ -250,7 +250,7 @@ impl FunctionTargetProcessor for BorrowAnalysisProcessor {
             BorrowAnnotation(BTreeMap::new())
         } else {
             let func_target = FunctionTarget::new(func_env, &data);
-            let mut analyzer = BorrowAnalysis::new(&func_target);
+            let analyzer = BorrowAnalysis::new(&func_target);
```

Comment:
The mut here was intended to allow the Analysis to accumulate some state, but I'm fine with removing it for now.